.github/workflows/release.yml

```diff
@@ -170,7 +170,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v1
         with:
-          node-version: 10.x
+          node-version: 14
 
       - name: Download Cached Deps
```

Comment:
Let's remove the cached dependencies step, so we don't pollute the cache for the other jobs.